### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by KeinR** [Mattiwos/SpaceclusterX#33]
This project looks interesting, however I see that the last commit was 4 months ago...

**Comment by whiskersthecat** [comment]
Yeah, we haven't worked on it in a while. We sort of ran out of motivation, although the project currently functions somewhat. Do you have any feedback or things that would be cool to see in the game? If so, we can work on implementing them. Thanks for taking a look.
